src/App.css

```diff
@@ -13,7 +13,93 @@
 .form-control:-webkit-autofill,
 :-webkit-autofill:active {
   transition: background-color 5000s;
-  -webkit-text-fill-color: white !important;
+  /* -webkit-text-fill-color: white !important; */
   background-color: transparent !important;
   outline: 0 !important;
 }
+
+body{
+  font-size: 15px;
```

Comment:
We shouldn't be using pixels for font-sizes this is not good for accessibility as users will not be able to increase it from their browsers settings.

You can read more about that in this article:
https://www.joshwcomeau.com/css/surprising-truth-about-pixels-and-accessibility/